app/views/cookie_preferences/_error_message.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if flash[:error] %>
```

Comment:
https://github.com/DFE-Digital/find-teacher-training/blob/d75a57aa84f0437fc44b3ddb2b91d59a8ee9a816/app/views/result_filters/_error_message.html.erb

what's the diff? Looks very same.

Reply:
Good shout out. I've refactored it to reuse it.